src/driftpy/accounts/oracle.py

```diff
@@ -95,8 +106,47 @@ def decode_pyth_price_info(
     )
 
 
+def decode_swb_price_info(data: bytes):
+    file = Path(str(driftpy.__path__[0]) + "/idl/switchboard.json")
+    with file.open() as f:
+        raw = file.read_text()
```

Comment:
i wonder if there is a way to avoid doing this eveytime this function is used .... can we have a static variable or something? otherwise might make sense to create oracle clients